website/package.json

```diff
@@ -12,7 +15,7 @@
     "rename-version": "docusaurus-rename-version"
   },
   "dependencies": {
-    "docusaurus": "1.0.0-beta.4"
+    "docusaurus": "^1.0.0-beta.4"
```

Comment:
Wasn't this locked on purpose https://github.com/facebook/jest/pull/4924? I see Yarn resolves this to 1.0.0-beta.4 anyway, but not sure why

Reply:
I was fancy and ran yarn first, then changed it manually. 🚂

Reply:
oh you!